src/js/components/location/LocationChooser/LocationChooserModal/LocationChooserModal.jsx

```diff
@@ -80,28 +81,30 @@ const LocationChooserModal = ({
       onRequestClose={onClose}
       portalClassName="location-chooser__modal"
     >
-      <header
-        className="location-chooser__modal__header d-flex flex-row justify-content-between align-items-center"
-      >
-        <h1 className="location-chooser__modal__header__title m-0">
-          <Translate
-            id="react.dashboard.chooseLocation.label"
-            defaultMessage="Choose Location"
-          />
-        </h1>
-        <button
-          onClick={onClose}
-          type="button"
-          className="location-chooser__modal__header__close-button"
+      <div data-testid="location-chooser-modal">
```

Comment:
Added a wrapper here because it wasn't possible to add any other attributes on `Modal` component